### ISSUE TITLE
Extension is unavailable in VSCodium

### ISSUE THREAD

**Issue by jeremypenner** [midouest/vscode-playdate-debug#37]
Just spun up my first Playdate project and this extension makes the whole development process extremely smooth, so thanks for that!

I was wondering if you'd consider publishing it to the open-vsx.org as well? I use VSCodium on Linux, which is prevented by Microsoft's terms of service from accessing Microsoft's extension marketplace. (You can hack it to do it anyway, which is how I tried this out, but it's not a great experience.) open-vsx seems to be the standard open extension repository for non-Microsoft builds of VS Code; my Linux distro has multiple different VS Code-derived packages that use it.

https://github.com/eclipse/openvsx/wiki/Publishing-Extensions has the details, but once you're set up with an account and an access token, it looks like it should be as straightforward as using `ovsx publish` alongside `vsce publish`.

Thanks!

**Comment by midouest** [comment]
Thanks for the suggestion! I will look into this.

It looks like another option is to make a .vsix bundle available for download. Do you install any extensions using a .vsix?

Someday I would like to automate publishing this extension to the marketplace. As part of that, I will probably automate publishing a .vsix bundle to the GitHub releases for this repository.

**Comment by Zoomll** [comment]
I know that this is an old issue, but are there any updates on this? There are still no Playdate extensions on open-vsx.org.

**Comment by cannikin** [comment]
@Zoomll If you install the plugin first on VS Code, you can right-click and "Download VSIX":

<img width="416" alt="Image" src="https://github.com/user-attachments/assets/2644b946-875d-4007-b342-01464cf7b25f" />

Then open the Command Palette in the other IDE and search for "Extensions: Install from VSIX..."

<img width="597" alt="Image" src="https://github.com/user-attachments/assets/d58f2d12-08ba-4f8b-9ab2-300c4eacbaa9" />

**Comment by Zoomll** [comment]
I understand that you can download the plugin like that, but then it won't auto update.

**Comment by cannikin** [comment]
That's true...is there a way to get notified of new releases via GitHub? You could wait for that email and do the install again...better than nothing, I guess.

**Comment by Zoomll** [comment]
I think there's a way to get notified of new releases, but it'd probably be best for for the creator of the plugin to upload it to Open VSX.

**Comment by midouest** [comment]
The extension is now published to Open VSX: https://open-vsx.org/extension/midouest/playdate-debug
